content/docs/cdn-links.md

```diff
@@ -1,37 +1,37 @@
 ---
 id: cdn-links
-title: CDN Links
+title: CDN लिंकस
 permalink: docs/cdn-links.html
 prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+React एवं ReactDOM दोनों CDN पर उपलब्ध हैं। 
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+उपरोक्त वर्शन केवल डेवलपमेंट के लिए हैं, एवं प्रोडक्शन के लिए नहीं। React के minified एवं ऑप्टीमाइज़्ड प्रोडक्शन वर्शन निम्नलिखित लिंक पर उपलब्ध हैं:
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-To load a specific version of `react` and `react-dom`, replace `16` with the version number.
+`react` एवं `react-dom` के विशिष्ट वर्शन लोड करने के लिए `16` की जगह वर्शन संख्या डाल दें। 
```

Comment:
डाल दें। -> के साथ बदलें

Reply:
Hi @saranshkataria,

`react` एवं `react-dom` के विशिष्ट वर्शन लोड करने के लिए `16` की जगह वर्शन संख्या के साथ बदलें sounds a bit odd to me. 
Maybe we could change it to:
`react` एवं `react-dom` के विशिष्ट वर्शन लोड करने के लिए `16` को वर्शन संख्या द्वारा बदल दें

Reply:
sounds good